src/trajectory_data_handling/sqlite_querying.cpp

```diff
@@ -0,0 +1,93 @@
+#include "sqlite_querying.hpp"
+
+namespace SQLite_Querying {
```

Comment:
Det er C++ convension:

```suggestion
namespace sqlite_querying {
```

Reply:
done